workflows/ensemblrepeatdownload.nf

```diff
@@ -45,17 +40,35 @@ include { CUSTOM_DUMPSOFTWAREVERSIONS } from '../modules/nf-core/modules/custom/
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 */
 
-workflow ENSEMBLDOWNLOAD {
+workflow ENSEMBLREPEATDOWNLOAD {
 
     ch_versions = Channel.empty()
 
-    //
-    // SUBWORKFLOW: Read in samplesheet, validate and stage input files
-    //
-    INPUT_CHECK (
-        ch_input
+    PARAMS_CHECK (
+        [
+            params.input,
+            params.assembly_accession,
+            params.ensembl_species_name,
+            params.outdir,
+        ]
+    )
```

Comment:
Same as https://github.com/sanger-tol/ensemblgenedownload/pull/1#discussion_r985810750

> Pass input as a single channel instead of the params directly.

Reply:
Addressed the same way as in ensemblgenedownload. The CLI params are bundled into a single channel, separately from the sample-sheet